bin/metrics-mesos.rb

```diff
@@ -33,6 +33,10 @@
 require 'socket'
 require 'json'
 
+# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
+MASTER_DEFAULT_PORT = '5050'
+SLAVE_DEFAULT_PORT = '5051'
```

Comment:
We should be using the default: attribute that options provide in sensu-plugin here.